packages/office-ui-fabric-react/src/components/Toggle/Toggle.base.tsx

```diff
@@ -146,8 +146,8 @@ export class ToggleBase extends React.Component<IToggleProps, IToggleState> impl
         {(keytipAttributes: any): JSX.Element => renderPill(keytipAttributes)}
       </KeytipData>
     ) : (
-        renderPill()
```

Comment:
Please could you revert this change? It's causing the check for changes in that package and fails build?

Reply:
Will do. It must have been some auto-magic on my machine as I did not touch these files :)